docs/source/development-guide/style-guide/naming-conventions.rst

```diff
@@ -19,23 +19,33 @@ The general filename convention is as follows::
 
 * ``<datalevel>`` is the data level for the data product.  Acceptable values depend on the instrument:
 
-  * ``codice``: ``l0``, ``l1a``, ``l1b``, ``l2``, ``l3``
-  * ``glows``: ``l0``, ``l1a``, ``l1b``, ``l2``, ``l3a``, ``l3b``, ``l3c``, ``l3d``
-  * ``hi-45``: ``l0``, ``l1a``, ``l1b``, ``l1c``, ``l2``, ``l3``
-  * ``hi-90``: ``l0``, ``l1a``, ``l1b``, ``l1c``, ``l2``, ``l3``
-  * ``hit``: ``l0``, ``l1a``, ``l1b``, ``l2``, ``l3``
-  * ``idex``: ``l0``, ``l1a``, ``l1b``, ``l1c``, ``l2``, ``l3``
-  * ``lo``: ``l0``, ``l1a``, ``l1b``, ``l1c``, ``l2``
-  * ``mag``: ``l0``, ``l1a``, ``l1b``, ``l1c``, ``l2pre``, ``l2``
-  * ``swapi``: ``l0``, ``l1``, ``l2``, ``l3a``, ``l3b``
-  * ``swe``: ``l0``, ``l1a``, ``l1b``, ``l2``, ``l3``
-  * ``ultra-45``: ``l0``, ``l1a``, ``l1b``, ``l1c``, ``l1d``, ``l2``, ``l3``
-  * ``ultra-90``: ``l0``, ``l1a``, ``l1b``, ``l1c``, ``l1d``, ``l2``, ``l3``
+  * CoDICE: ``l0``, ``l1a``, ``l1b``, ``l2``, ``l3``
+  * GLOWS: ``l0``, ``l1a``, ``l1b``, ``l2``, ``l3a``, ``l3b``, ``l3c``, ``l3d``
+  * HIT: ``l0``, ``l1a``, ``l1b``, ``l2``, ``l3``
+  * IDEX: ``l0``, ``l1a``, ``l1b``, ``l1c``, ``l2``, ``l3``
+  * IMAP-Hi: ``l0``, ``l1a``, ``l1b``, ``l1c``, ``l2``, ``l3``
+  * IMAP-Lo: ``l0``, ``l1a``, ``l1b``, ``l1c``, ``l2``
+  * IMAP-Ultra: ``l0``, ``l1a``, ``l1b``, ``l1c``, ``l1d``, ``l2``, ``l3``
+  * MAG: ``l0``, ``l1a``, ``l1b``, ``l1c``, ``l2pre``, ``l2``
+  * SWAPI: ``l0``, ``l1``, ``l2``, ``l3a``, ``l3b``
+  * SWE: ``l0``, ``l1a``, ``l1b``, ``l2``, ``l3``
 
 * ``<descriptor>`` stores information specific to the instrument and can store any information that is relevant. For
   example, it will typically contain the data product name, and additionally may contain the pointing number (e.g.
   ``burst-7`` for 'burst' mode and the 7th repointing). This field can have any text or numbers as long as it doesn't
-  include underscores.
+  include underscores. The following is a list of expected descriptors for each instrument:
+
+  * CoDICE: ``hi``, ``lo``
+  * GLOWS: TBD
+  * HIT: TBD
+  * IDEX: TBD
+  * IMAP-Hi: TBD
+  * IMAP-Lo: TBD
+  * IMAP-Ultra: TBD
+  * MAG: TBD
+  * SWAPI: TBD
+  * SWE: TBD
```

Comment:
If anyone knows the descriptors that will be used for the instrument(s) they are working on and could comment here that would be helpful!